### PR TITLE
Add the SDK initialization docs to READMEs

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,79 @@ To update native dependencies to latest available versions, run the following co
 
 [CLI autolink feature](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md) links the module while building the app.
 
+### Initializing 
+
+Initializing the SDK should be done in native code inside your `AppDelegate` (iOS) and `MainApplication` (Android). The following code snippets demonstrate how to import and register the Mobile Core, Identity, Lifecycle, Signal, and Profile extensions. For other extensions, the documentation on how to initialize the extension can be found in `./packges/{extension}/README.md`
+
```
Comment (cacheung) [inline README.md:56]: typo for packages

File: README.md [hunk 2]
```diff
@@ -50,6 +50,79 @@ To update native dependencies to latest available versions, run the following co
...
+    try {
+      UserProfile.registerExtension();
+      Identity.registerExtension();
+      Lifecycle.registerExtension();
+      Signal.registerExtension();
+      // register other extensions here
+      MobileCore.start(new AdobeCallback () {
+          @Override
+          public void call(Object o) {
+            MobileCore.configureWithAppID("your-app-ID");
```
Comment (cacheung) [inline README.md:116]: nit: should we have the same code as Sample App?  Or we can change the Sample App to match this.

File: packages/core/README.md [hunk 3]
```diff
@@ -5,9 +5,9 @@
 
 ## Usage
 
-### Initializing:
+### Initializing and registering the extension
 
-Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk#2-add-initialization-code). The linked documentation initializes the User Profile extension which is not required, however, if you are interested in the User Profile extension for React Native you can find it [here](https://www.npmjs.com/package/@adobe/react-native-aepuserprofile).
+Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](../../README.md#initializing).
```
Comment (cacheung) [inline packages/core/README.md:10]: Will this readme be the main page in npm? If so let use the absolute link of the Beta home readme page? https://github.com/adobe/aepsdk-react-native

File: packages/userprofile/README.md [hunk 4]
```diff
@@ -6,6 +6,10 @@
 
 ## Usage
 
+### Initializing and registering the extension
+
+Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](../../README.md#initializing).
+
```
Comment (cacheung) [inline packages/userprofile/README.md:12]: Use absolute link?

File: README.md (hunk 2 above)
Comment (yangyansong-adbe) [reply README.md:116]: Sure. I will update it as what we have in the Sample app.

File: packages/core/README.md (hunk 3 above)
Comment (yangyansong-adbe) [reply packages/core/README.md:10]: Good catch!

**Comment by yangyansong-adbe** [comment]
@cacheung your review comments have been addressed.

**Review by cacheung — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -50,6 +50,79 @@ To update native dependencies to latest available versions, run the following co
...
+    MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG);
+    MobileCore.setWrapperType(WrapperType.REACT_NATIVE);
+
+    try {
+      UserProfile.registerExtension();
+      Identity.registerExtension();
+      Lifecycle.registerExtension();
+      Signal.registerExtension();
+      MobileCore.configureWithAppID("yourAppID");
```
Comment (cacheung) [inline README.md:112]: Should we also add
 // register other extensions here
 as iOS

**Pull request merged**